src/Plugin/Field/FieldFormatter/GoogleFormFormatter.php

```diff
@@ -55,6 +78,7 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
           'src' => $url,
           'title' => $iframe_title,
           'class' => ['google-form'],
+          'style' => 'height: ' . $iframe_height . 'px;',
```

Comment:
Why not just use a `height` attribute instead of inline styles?

Reply:
That's an a11y violation.  The height attribute is deprecated, inline styles are the proper method.

Reply:
oh yeah. I recall that now.

Reply:
Changed it to use the attribute.